moment-element.html

```diff
@@ -33,80 +33,68 @@
          * For consistent results, parsing anything other than ISO 8601 strings
          * with the `inputFormat` property. More information in [moment String](http://momentjs.com/docs/#/parsing/string/).
          */
-        datetime: String,
+        datetime: {
+          type: String,
+          value: function() {
+            return new Date().getTime();
```

Comment:
The "datetime" variable implies that you are getting the date's time.

Should we change it to type Number to enforce?
Should we stay lenient since moment accepts Number and String?
Should we rename the variable to date?